exploration/default-registry-and-mf1-compatibility.md

```diff
@@ -164,27 +167,28 @@ The function `:date` has these function-specific _style_ options:
   - `full`
   - `long`
   - `medium`
-  - `short`
+  - `short` (default)
 
 The function `:time` has these function-specific _style_ options:
 - `style`
   - `full`
   - `long`
   - `medium`
-  - `short`
+  - `short` (default)
 
-The _field_ options are defined as follows:
+##### Field Options
 
-All functions have the following option:
-- `timeZoneName`
-  - `long`
-  - `short`
-  - `shortOffset`
-  - `longOffset`
-  - `shortGeneric`
-  - `longGeneric`
+Field options describe which fields to include in the formatted output
+and what format to use for that field.
+Only those fields specified in the _annotation_ appear in the formatted output.
```

Comment:
One of the features of semantic skeleta is preventing nonsensical skeletons like "hours and seconds" without minutes. Mapping from standard skeleta to semantic skeleta may fill in some of those missing fields. Even now, fields such as hour cycle and era may appear even if not in the skeleton. Therefore, it would be better to remove the second sentence giving any guarantees about which fields appear in the formatted output.